### PR TITLE
Better argument list

### LINKED ISSUE #37: Ugly object instantiation (and maybe other method calls?)
Hi, prettier-java gives me this:
```java
  @Override
  public ScatterChart scatterChart(
    DbCategorisationLevel grouping,
    FactorisedSelection selection,
    double marginRateTarget
  ) {
    return new DbScatterChart(context,
    x,
    y,
    grouping,
    new FactorisedFilterCondition(schema,
    selection),
    marginRateTarget);
  }
```

I would have expected something like this:
```java
  @Override
  public ScatterChart scatterChart(
    DbCategorisationLevel grouping,
    FactorisedSelection selection,
    double marginRateTarget
  ) {
    return new DbScatterChart( // if some of the parameters are wrapped, then all should even the first one
      context, // the parameters should be indented of one indentation
      x,
      y,
      grouping,
      new FactorisedFilterCondition( // idem
        schema, // idem
        selection
      ), // I find it really easier to read with the parenthesis to the line when parameters are wrapped
      marginRateTarget
   ); // idem
  }
```

Or if there was enough space:
```java
  @Override
  public ScatterChart scatterChart(
    DbCategorisationLevel grouping,
    FactorisedSelection selection,
    double marginRateTarget
  ) {
    return new DbScatterChart(
      context,
      x,
      y,
      grouping,
      new FactorisedFilterCondition(schema, selection),
      marginRateTarget
   );
  }
```

### PR DESCRIPTION
**Pull request by victornoel**
Closes #37 and #39.

### CODE REVIEW

File: PREVIEW.md [hunk 1]
```diff
@@ -1342,9 +1428,9 @@ public class Variables {
         int castVariable = (int) (4 / 2);
     }
 
-    public void localVariables(
-
-    ) {final List<Filter> filterList = new ArrayList<>();
+    public void localVariables() {final List<
```
Comment (thorbenvh8) [inline PREVIEW.md]: the body should start in a new line

Comment (victornoel) [reply PREVIEW.md]: see #38, I already reported the problem :)

Comment (thorbenvh8) [reply PREVIEW.md]: ok im gonna merge, I can live with this then, and we can get it into master. It's not worth then before so we should merge it :)

**Comment by thorbenvh8** [comment]
Really nice, thank you!

**Pull request merged**